src/sentry/tasks/post_process.py

```diff
@@ -1322,11 +1325,53 @@ def should_postprocess_feedback(job: PostProcessJob) -> bool:
     return False
 
 
+def get_project_counts(project: Project) -> int:
+    # Dummy function, we'll replace this in a later pr.
+    return 1000
+
+
+MAX_NEW_ESCALATION_AGE_HOURS = 24
+
+
+def detect_new_escalation(job: PostProcessJob):
+    """
+    Detects whether a new issue is escalating. New issues are issues less than
+    MAX_NEW_ESCALATION_AGE_HOURS hours old.
```

Comment:
Not particularly related to this PR, but we have a NEW substatus that means < 3 days old.

I'm wary about adding a second definition for 'new' especially related to escalating which is so closely tied to the issue states. Any way we can keep this from getting confusing down the line?